src/docs/frontend/component-library.mdx

```diff
@@ -138,6 +138,16 @@ Helper Components:
     />
     ```
 
+- [`<ThemeToggle />`](https://github.com/getsentry/sentry/blob/master/static/app/components/stories/themeToggle.tsx)<br />
+    A wrapper component to isolate children from the current theme used on the page. Allows quickly toggling an example between <code>light</code>/<code>dark</code> without changing the whole screen (useful at night when you want most things in dark mode).<br/>
+    Use this to emphasize that a component/image has been tested and works well in <code>light</code> or <code>dark</code> mode.
+    ```typescript
```

Comment:
small nit: I think `tsx` would highlight this better

Reply:
Yup! much better

| typescript | tsx |
| --- | --- |
| <img width="723" alt="SCR-20240108-kdea" src="https://github.com/getsentry/develop/assets/187460/8ac79730-3c1b-450c-a2ee-8b3ccff9a0dc"> | <img width="724" alt="SCR-20240108-kddj" src="https://github.com/getsentry/develop/assets/187460/298f78e9-b39f-43d5-8731-b8c2351feb66">